tests/admin_checks/tests.py

```diff
@@ -138,6 +143,86 @@ class SongAdmin(admin.ModelAdmin):
         ]
         self.assertEqual(errors, expected)
 
+    def test_wrong_prepopulated_fields(self):
+        class SongAdmin(admin.ModelAdmin):
+            prepopulated_fields = {'title': 'test'}
+
+        expected = [
+            checks.Error(
+                'The value of \'prepopulated_fields["title"]\' must be a list'
+                ' or tuple.',
+                obj=SongAdmin,
+                id='admin.E029',
+            )
+        ]
+        self.assertEqual(SongAdmin(Song, AdminSite()).check(), expected)
+
+    def test_wrong_readonly_fields(self):
+        class SongAdmin(admin.ModelAdmin):
+            readonly_fields = 'test'
+
+        expected = [
+            checks.Error(
+                "The value of 'readonly_fields' must be a list or tuple.",
```

Comment:
Okay, it's just since you used single-quotes in the upper test and escaped them in the string I though you might want to do it everywhere.